GUI/PyQt/framework1.py

```diff
@@ -11,102 +11,163 @@
 class Ui_MainWindow(object):
     def setupUi(self, MainWindow):
         MainWindow.setObjectName("MainWindow")
-        MainWindow.resize(1340, 849)
+        MainWindow.resize(1796, 979)

```

Comment:
What if the screen is smaller than the defined size? Can you automatically determine the maximal possible size?
Is there a desirable height/width ratio of the GUI which is looking good or does it work for any ratio?